### ISSUE TITLE
Select Component not tabbable on Safari

### ISSUE THREAD

**Issue by senadir** [WordPress/gutenberg#31189]
On Safari, the Select Component (which is a button) is not tabbable, as in, if you tab around, it will get ignored.

I know you can enable this on the browser, but there should be a way for it to work by default.

https://user-images.githubusercontent.com/6165348/116076656-6e3c2600-a68c-11eb-80b0-48b6593ad0ca.mov

By comparison, a default select component is selectable.

https://codepen.io/assassinateur/pen/NWdJrzG

**Comment by mirka** [comment]
This doesn't seem to reproduce anymore. Currently the `SelectControl` component renders an actual `<select>`, and is tabbable in Safari.

Let us know if this isn't the case!
